integration/resource_lacework_integration_service_now_test.go

```diff
@@ -0,0 +1,73 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestServiceNowRestAlertChannelCreate applies integration terraform '../examples/resource_lacework_alert_channel_service_now'
+// Uses the go-sdk to verify the created integration
+// Applies an update with new channel name and Terraform destroy
+func TestServiceNowRestAlertChannelCreate(t *testing.T) {
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/resource_lacework_alert_channel_service_now",
+		Vars: map[string]interface{}{
+			"channel_name":         "Service Now Alert Channel Example",
+			"instance_url":         "https://dev123.service-now.com",
+			"username":             "snow-user",
+			"password":             "snow-pass",
+			"custom_template_file": customTemplate,
+		},
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Create new ServiceNowRest Alert Channel
+	create := terraform.InitAndApplyAndIdempotent(t, terraformOptions)
+	created := GetAlertChannelProps(create)
+	data, ok := created.Data.Data.(map[string]interface{})
+	assert.True(t, ok)
+
+	actualName := terraform.Output(t, terraformOptions, "channel_name")
+	actualUrl := terraform.Output(t, terraformOptions, "instance_url")
+	actualUsername := terraform.Output(t, terraformOptions, "username")
+	actualTemplate := terraform.Output(t, terraformOptions, "custom_template_file")
+
+	assert.Equal(t, "Service Now Alert Channel Example", created.Data.Name)
+	assert.Equal(t, "https://dev123.service-now.com", data["instanceUrl"])
+	assert.Equal(t, templateEncoded,
+		data["customTemplateFile"])
+	assert.Equal(t, "snow-user", data["userName"])
+
+	assert.Equal(t, "https://dev123.service-now.com", actualUrl)
+	assert.Equal(t, "snow-user", actualUsername)
+	assert.Equal(t, customTemplate, actualTemplate)
+
+	// Update ServiceNowRest Alert Channel
+	terraformOptions.Vars = map[string]interface{}{
+		"channel_name": "Service Now Alert Channel Updated"}
```

Comment:
Can we inject new fields for all the variables/inputs, that way we can test that they all changed correctly.